src/string_util.cc

```diff
@@ -24,13 +24,13 @@ static_assert(arraysize(kBigSIUnits) == arraysize(kBigIECUnits),
 static_assert(arraysize(kSmallSIUnits) == arraysize(kBigSIUnits),
               "Small SI and Big SI unit arrays must be the same size");
 
-static const int kUnitsSize = arraysize(kBigSIUnits);
+static const int64_t kUnitsSize = arraysize(kBigSIUnits);
```

Comment:
arraysize returns a size_t. These need to be 64 bits to prevent those exact warnings. Without these changes `-Wshorten-64-to-32` fires.